### PR TITLE
added console log args

### LINKED ISSUE #56: Add action arguments fields for console-log action
Depends on #54.

It just takes a single `message` field.

### PR DESCRIPTION
**Pull request by tiftran**
Fixes #56 

so i took a stab at this...

### CODE REVIEW

**Comment by tiftran** [comment]
my second attempt given the new actionargument json editor?

File: extension/content/components/recipes/form/arguments/ConsoleLog.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        onChange={handleChange}
+      />
+    </FormGroup>
+  );
+}
+
+ConsoleLog.propTypes = {
+  onChange: PropTypes.func,
+  value: PropTypes.object,
+};
```
Comment (rehandalal) [inline extension/content/components/recipes/form/arguments/ConsoleLog.js]: this isn't needed anymore

File: extension/content/components/recipes/form/arguments/ConsoleLog.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Input, FormGroup, ControlLabel } from "rsuite";
+import {
```
Comment (rehandalal) [inline extension/content/components/recipes/form/arguments/ConsoleLog.js]: nit: new line

**Review by rehandalal — CHANGES_REQUESTED** [review]
this looks great! couple little nits.

i'm just going to pull this down and make sure it works as expected but I'm sure it will.

nice work!

**Comment by rehandalal** [comment]
Oh and we probably want to use the `InputField` from #105

File: extension/content/components/recipes/form/arguments/ConsoleLog.js [hunk 3]
```diff
@@ -1,41 +1,6 @@
 import React from "react";
-import PropTypes from "prop-types";
-import { Input, FormGroup, ControlLabel } from "rsuite";
-import {
-  ACTION_UPDATE_DATA,
-  useRecipeDetailsData,
-  useRecipeDetailsDispatch,
-} from "devtools/contexts/recipeDetails";
+import InputField from "devtools/components/recipes/form/arguments/fields/InputField";
```
Comment (rehandalal) [inline extension/content/components/recipes/form/arguments/ConsoleLog.js:3]: nit: new line between module and application imports

**Review by rehandalal — APPROVED** [review]
small nit but r+wc

bors delegate+

**Pull request merged**